### DISCUSSION TITLE
is it possible to send more than 1KB of data with UDP?

### DISCUSSION THREAD

**Discussion by jg-spitfire** [dannagle/PacketSender#294] [Q&A] [answered]
Hi, I am testing UDP protocol, I am loading and sending files of different sizes and I realize that when the file is greater than 1KB the file is fully loaded in the GUI and in the log is shown that the entire data was send (for example 8KB), but in the other side of the connection I only receive 1316 or 1440 Bytes, why?

if I send 500 Bytes, for example, I can see in the rx terminal that 500 Bytes were received

**Comment by dannagle** [comment, marked as answer]
UDP typically fragments at 1500 bytes, the size of an ethernet frame. Above that sending is not reliable.
